pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+filterwarnings =
+  # ovh has apparently vendored an old requests version that hits warnings :-/, it'll go away/move soon enough
+  ignore::DeprecationWarning:ovh.*:
```

Comment:
No longer required given https://github.com/octodns/octodns/pull/853 is merged?

Reply:
Yup. I'm working my way back through this series now, starting with https://github.com/octodns/octodns/pull/850. 

I'm also going to make the changes @istr did over in https://github.com/octodns/octodns-constellix/pull/5/files#diff-b0ddc24027292e24c495fb6a8df798ce6f62bd9b2a89e23b97d56e041e29b5a1R41 around coverage support build into pytest.